index.bs

```diff
@@ -452,11 +386,79 @@ partial interface NeuralNetworkContext {
+    element-wise binary operation of the two input tensors.
+
+    The element-wise binary operation will be broadcasted according to
+    [[!numpy-broadcasting-rule]]. The rank of the output tensor is the maximum
+    rank of the input tensors. For each dimension of the output tensor, its size
+    is the maximum size along that dimension of the input tensors.
+</div>
+
+### element-wise unary operations ### {#api-neuralnetworkcontext-unary}
+Compute the element-wise unary operation for input tensor.
+<script type=idl>
+partial interface NeuralNetworkContext {
+  Operand abs(Operand x);
+  Operand ceil(Operand x);
+  Operand cos(Operand x);
+  Operand exp(Operand x);
+  Operand floor(Operand x);
+  Operand log(Operand x);
+  Operand neg(Operand x);
+  Operand sigmoid(Operand x);
+  Operand sin(Operand x);
+  Operand sqrt(Operand x);
+  Operand tan(Operand x);
+  Operand tanh(Operand x);
+};
+</script>
+<div algorithm=unary>
+    **Arguments:**
+        - *x*: an {{Operand}}. The input tensor.
+
+    **Returns:** an {{Operand}}. The output tensor that contains the result of
+    element-wise unary operation of the input tensor. The shape of the output
+    tensor is the same as the shape of input tensor.
+
+    **Operation types:**
+        - *abs*: Compute the absolute value of the input tensor, element-wise.
+        - *ceil*: Compute the ceiling of the input tensor, element-wise.
+        - *cos*: Compute the cosine of the input tensor, element-wise.
+        - *exp*: Compute the exponential of the input tensor, element-wise.
+        - *floor*: Compute the floor of the input tensor, element-wise.
+        - *log*: Compute the natural logarithm of the input tensor, element-wise.
+        - *neg*: Compute the numerical negative value of the input tensor, element-wise.
+        - *sigmoid*: Compute the sigmoid function of the input tensor, element-wise.
+        - *sin*: Compute the sine of the input tensor, element-wise.
+        - *sqrt*: Compute the square root of the input tensor, element-wise.
+        - *tan*: Compute the tangent of the input tensor, element-wise.
+        - *tanh*: Compute the hyperbolic tangent of the input tensor, element-wise.
+</div>
+
 ### gemm ### {#api-neuralnetworkcontext-gemm}
-Calculate the <a href="https://en.wikipedia.org/wiki/Basic_Linear_Algebra_Subprograms#Level_3">general matrixmultiplication of the Basic Liner Algebra Subprograms</a>. The calculation follows the expression `alpha * A * B + beta *C`, where `A`, `B`, and `C` are matrices, and `A` and `B` may optionally be transposed prior to the calculation.
+Calculate the [general matrixmultiplication of the Basic Liner Algebra Subprograms](https://en.wikipedia.org/wiki/Basic_Linear_Algebra_Subprograms#Level_3). The calculation follows the expression `alpha * A * B + beta *C`, where `A`, `B`, and `C` are matrices, and `A` and `B` may optionally be transposed prior to the calculation.
```

Comment:
Is having matrixmultiplication as one word intentional? Two words seems better to me.

Reply:
its a typo from the past. i'll fix it.